products/pdt/pdtbq/src/meta.rs

```diff
@@ -99,7 +99,7 @@ impl MetaTable for Meta {
         start: i64,
         nr_blks: i64,
     ) -> Result<Option<(i64, String)>> {
-        let query = format!("SELECT start_blk,nr_blks,client_id FROM {} WHERE start_blk <= {} and nr_blks >= {} - start_blk ORDER BY nr_blks DESC LIMIT 1",
+        let query = format!("SELECT start_blk,nr_blks,client_id FROM `{}` WHERE start_blk <= {} and nr_blks >= {} - start_blk ORDER BY nr_blks DESC LIMIT 1",
```

Comment:
This was returning an error whenever the meta-tables had to be accessed. Because it had numbers in the name it was causing parsing errors on BQ side.